### ISSUE TITLE
ZWED0159W - Plugin (org.zowe.zlux.proxy.zosmf) loading failed in logs 

### ISSUE THREAD

**Issue by Joe-Winchester** [zowe/zlux#420]
Users are seeing the message
```
ZWED0159W - Plugin (org.zowe.zlux.proxy.zosmf) loading failed.
```
in log files, e.g.
```
yyyy-mm-dd hh:mm:ss:ms ZWED:16843061 ZWESVUSR WARN (_zsf.install,index.js:255) 
ZWED0159W - Plugin (org.zowe.zlux.proxy.zosmf) loading failed. 
Message: "ZWED0047E - Proxy (org.zowe.zlux.proxy.zosmf:data) setup failed.
Host & Port for proxy destination are required but were missing.
```
This is now documented in troubleshooting at something users should ignore
Can this message be supressed so that it doesn't get written or is it a symptom of something that needs fixing ?

**Comment by 1000TurquoisePogs** [comment]
Should not be seen in 1.11.0. What users are seeing this?

**Comment by Joe-Winchester** [comment]
@1000TurquoisePogs - Great that this is fixed in 1.11 - I think the users in slack weren't at 1.11.  I created a doc PR  https://github.com/zowe/docs-site/pull/1207 to let users know that the troubleshooting guide now mentions the error is removed in 1.11
